src/api.ts

```diff
@@ -13,8 +15,9 @@ function wait(delay: number): Promise<void> {
 
 function get<T>(url: string): Promise<T> {
   // eslint-disable-next-line prefer-template
-  const fullURL = BASE_URL + url + '.json';
+  const fullURL = `${BASE_URL}${url}.json`;
```

Comment:
The `fullURL` is constructed by appending the endpoint to the `BASE_URL`. Ensure that the `BASE_URL` is correct and that the endpoints are accessible. If there are any issues with fetching data, verify the URL structure.